frame/election-provider-support/solution-type/src/single_page.rs

```diff
@@ -174,9 +174,9 @@ pub(crate) fn generate(def: crate::SolutionDef) -> Result<TokenStream2> {
 		}
 
 		type __IndexAssignment = _npos::IndexAssignment<
-			<#ident as _npos::NposSolution>::VoterIndex,
-			<#ident as _npos::NposSolution>::TargetIndex,
-			<#ident as _npos::NposSolution>::Accuracy,
+			<#ident as frame_election_provider_support::NposSolution>::VoterIndex,
```

Comment:
This is wrong. You now assume the existence of this module in scope. 

Look into how _npos is created, we need a similar approach. And ideally we remove _npos as it is nota used much anymore. 

(sorry for the short feedback, hopefully it is clear, afk now)

Reply:
Implemented it better, I had to do a few more changes to completely remove the dependency on _npos